pom.xml

```diff
@@ -28,70 +28,50 @@
         <commons-compress.version>1.10</commons-compress.version>
         <commons-net.version>3.4</commons-net.version>
         <commons-io.version>2.4</commons-io.version>
         <commons-codec.version>1.10</commons-codec.version>
-        <commons-collections.version>3.2.1</commons-collections.version>
         <!-- TODO: [ES] Validate.notNull() behaviour has changed in commons-lang3 (throws NullPointerException) -->
         <commons-lang.version>2.6</commons-lang.version>
         <commons-validator.version>1.5.1</commons-validator.version>
         <commons-ip-math.version>1.23</commons-ip-math.version>
-
         <diffutils.version>1.3.0</diffutils.version>
-
         <ezvcard.version>0.8.2</ezvcard.version>
-
         <findbugs.version>1.3.9</findbugs.version>
-
         <guava.version>19.0</guava.version>
-
         <!-- TODO: [ES] hazelcast 3.6 loses data on node shutdown. See hazelcast branch -->
         <hazelcast.version>2.6.10</hazelcast.version>
         <hamcrest.version>1.3</hamcrest.version>
-        <httpclient.version>4.5.2</httpclient.version>
-
+        <httpclient.version>4.5.3</httpclient.version>
         <jaxb.version>2.2.11</jaxb.version>
         <jflex.version>1.6.1</jflex.version>
-        <jodatime.version>2.9.2</jodatime.version>
+        <jodatime.version>2.9.9</jodatime.version>
         <jopt-simple.version>4.8</jopt-simple.version>
-        <jsoup.version>1.8.3</jsoup.version>
+        <jsoup.version>1.10.3</jsoup.version>
         <junit.version>4.12</junit.version>
-        <jetty.version>9.3.12.v20160915</jetty.version>
-        <!-- TODO: [ES] update Jersey to 2.16 (creates unmarshalling problems with crowd) -->
-        <jersey.version>2.11</jersey.version>
-        <jackson.version>2.7.2</jackson.version>
-
+        <jetty.version>9.4.7.v20170914</jetty.version>
+        <jersey.version>2.25.1</jersey.version>
+        <jackson.version>2.9.2</jackson.version>
         <!-- TODO: [ES] apache-log4j-extras 1.2.17 duplicates classes from log4, causing warnings from surefire -->
         <log4j.version>1.2.17</log4j.version>
         <lucene.version>4.10.4</lucene.version>
-
-        <!-- TODO: [ES] javax.mail-api 1.5.2 causes gmaven plugin to fail -->
-        <mail.version>1.4.5</mail.version>
-        <mariadb-java-client.version>1.2.0</mariadb-java-client.version>
+        <mail.version>1.6.0</mail.version>
+        <mariadb-java-client.version>2.1.2</mariadb-java-client.version>
         <mockito.version>1.9.5</mockito.version>
-
-        <!-- TODO: [ES] netty 3.10.0 creates excessive threads during integration tests -->
-        <netty.version>3.4.6.Final</netty.version>
-
+        <netty.version>3.10.6.Final</netty.version>
         <persistence.version>1.0.2</persistence.version>
-
         <slf4j.version>1.7.18</slf4j.version>
-        <spring.version>4.3.2.RELEASE</spring.version>
-        <springockito.version>1.0.5</springockito.version>
+        <spring.version>5.0.1.RELEASE</spring.version>
```

Comment:
Do we have to upgrade to Spring 5 to fix security issues? Is there a patched 4.x release we can use instead? Spring 5 may contain a large amount of new features, with potential new issues.